cdap-docs/developer-manual/source/pipelines/developing-plugins/presentation-plugins.rst

```diff
@@ -421,6 +421,71 @@ CDAP pipelines as of version |version|.
             }
           }
      
+   * - ``multi-select``
+     - - ``delimiter``: delimiter to concatenate multiple values to form a single string
+       - ``options``: array of options where each option is defined by
+         - ``id``: an identifier to uniquely id an option
+         - ``label``: a label to be rendered in UI for that specific option
+       - ``defaultValue``: an array of default value(s) to use when rendering the widget for the first time
+     - ``string``
+     - Multi-select dropdown to choose one or more from a list of available options.
+     - .. container:: copyable copyable-text
+
+         ::
+
+          {
+            "name": "property-keyvalue",
+            "widget-type": "multi-select",
+            "widget-attributes": {
+              "delimiter": ",",
+              "defaultValue": [
+                "value1",
+                "value2"
+              ],
+              "options": [
+                {
+                  "id": "value1",
+                  "label": "Value One"
+                },
+                {
+                  "id": "value2",
+                  "label": "Value two"
+                }
+              ]
+            }
+          }
+     
+   * - ``radio-group``
+     - - ``layout``: Layout for the radio buttons to render. Defaults to `block` to render one below the other. Another option is `inline` to render side-by-side
+       - ``options``: an array of options where each option is defined by
+         - ``id``: an identifier to uniquely id an option
```

Comment:
I think this should say the value that is going to be set. It's not just for an identifier